src/components/ha-form/types.ts

```diff
@@ -29,6 +29,7 @@ export interface HaFormBaseSchema {
 export interface HaFormGridSchema extends HaFormBaseSchema {
   type: "grid";
   name: "";
+  min_width?: string;
```

Comment:
This name needs to specify that it's the min width of "what". The whole grid or a single cell